doc/classes/Vector2.xml

```diff
@@ -110,7 +110,8 @@
 			<return type="float" />
 			<argument index="0" name="with" type="Vector2" />
 			<description>
-				Returns the cross product of this vector and [code]with[/code].
+				Returns the 2D analog of the cross product for this vector and [code]with[/code].
+				[b]Note:[/b] Cross product is not defined in 2D mathematically. This method embeds the 2D vectors in the XY plane of 3D space and uses their cross product's Z component as the analog.
```

Comment:
```suggestion
				Returns the 2D analog of the cross product for this vector and [code]with[/code]. This is the signed area of the parallelogram formed by the two vectors. If the second vector is clockwise of the first vector, then the area is positive. If counter-clockwise, the area is negative.
```

I'm not sure that the `[b]Note:[/b]` part is helpful. Do Godot users care about how something is originally defined by mathematicians for usage in 3D, or do they care about how it's defined in 2D and in Godot? We are already saying that Vector2's cross method is the closest analog to the cross product in 2D.

The cross product in 2D is an extension of the original 3D definition, losing the property of the resulting vector being perpendicular to both input vectors (which is impossible in 2D unless the two vectors are colinear but then the cross product is zero anyway). It's the closest analog that exists.

We could also note for Vector3 that the magnitude of the cross product vector is the area of the parallelogram formed by the two vectors.